python/GafferArnoldUI/ArnoldOptionsUI.py

```diff
@@ -490,6 +492,17 @@ def __gpuSummary( plug ) :
 
 		],
 
+		"options.dielectricPriorities" : [
+
+			"description",
+			"""
+			Enable or disable dielectric priorities for surface shaders
```

Comment:
We're missing a full stop at the end, and using the imperative "Enable" when the rest of the docs use "Enables" as in "This enables".